### PR TITLE
Default enarx trust process

### PR DESCRIPTION
**Pull request by MikeCamel**
Changes based on @ueno's feedback.

### CODE REVIEW

File: default-enarx-trust-process.md [hunk 1]
```diff
@@ -0,0 +1,478 @@
...
+   - loads **Enarx runtime instance** into **TEE instance**
+   - creates a Main loop component
+ - Main loop
+   - one per Keep
+   - lifespan coterminous with the Keep it services
+   - provides the syscall processing from outside the Keep
+
+The initialisation of the process is prompted by the **Orchestrator**, which
+communicates with the **Enarx client agent**, which contacts the **Enarx
+host agent** over a network connection (which does not need to be
```
Comment (lkatalin) [inline default-enarx-trust-process.md:144]: This paren is still unbalanced. : (

File: default-enarx-trust-process.md [hunk 2]
```diff
@@ -0,0 +1,478 @@
...
+instance and Enarx runtime image within it to an Enarx Keep.  This
+transition is considered complete once the Keep has been attested.
+
+The measurement of the TEE instance and Enarx runtime image within it
+is requested by the **Enarx host agent** and performed by the **host
+CPU + CPU firmware**.  The measurement is then transmitted from the
+**Enarx host agent** to the **Enarx client agent**.
+
+The **Enarx client agent** checks the attestation measurement database
+(see (Issue #25)[https://github.com/enarx/rfcs/issues/25]) to ensure
```
Comment (lkatalin) [inline default-enarx-trust-process.md:241]:
```suggestion
(see [Issue #25](https://github.com/enarx/rfcs/issues/25)) to ensure
```

**Comment by lkatalin** [comment]
I'm not blocking on this, but there are a couple remaining typos.

**Comment by axelsimon** [comment]
I tried to incorporate the latest changes to this and rebase Mike's branch on upstream, via a PR to his personal repo, but it didn't work.

At this point, to simplify, i strongly suggest closing this PR and using #44 instead, which i just created, as a fresh start.
To that end, i'm closing tihs P.

**Pull request closed without merging**